templates/publisher/listing.html

```diff
@@ -206,9 +205,8 @@ <h2 class="p-heading--four">
           <label for="video" class="p-form__label">Video:</label>
         </div>
         <div class="col-8">
-          <div class="p-form__control">
+          <div class="p-form__control" data-tour="listing-video">
```

Comment:
Moving some tour elements around do make sure whole interactive area is highlighted and available during the tour.